ext/LogExpFunctionsChainRulesCoreExt.jl

```diff
@@ -187,4 +187,9 @@ end
 ChainRulesCore.@scalar_rule(cloglog(x), (-inv((1 - x) * log1p(-x)),))
 ChainRulesCore.@scalar_rule(cexpexp(x), (-xexpx(-exp(x)),))
 
+ChainRulesCore.@scalar_rule(loglogistic(x::Real), (logistic(-x),))
+ChainRulesCore.@scalar_rule(log1mlogistic(x::Real), (-logistic(x),))
+ChainRulesCore.@scalar_rule(logitexp(x::Real), (inv(1 - exp(x)),))
+ChainRulesCore.@scalar_rule(logit1mexp(x::Real), (-inv(1 - exp(x)),))
+
```

Comment:
IMO these are not needed - AD handles the functions already fine since rules for `log1pexp` etc. are defined.
```suggestion
```

Reply:
As directed, these definitions have been removed.

Reply:
Looking through the issues, [#36](https://github.com/JuliaStats/LogExpFunctions.jl/issues/36#issue-1160600914) seems to indicate that these functions should support `ChainRulesCore` forward and reverse rules.